dbt_server/helpers.py

```diff
@@ -15,3 +17,19 @@ def extract_compiled_code_from_node(result_node_dict):
         raise InternalException(msg)
 
     return compiled_code
+
+
+def set_profile_name(args=None):
+    # If no profile name is passed in args, we will attempt to set it from env vars
+    # If no profile is set, dbt will default to reading from dbt_project.yml
+    if args and args.profile:
+        return args
+    if os.getenv("DBT_PROFILE_NAME"):
+        if args == None:
+
+            class Args(BaseModel):
```

Comment:
why is type being defined inside this if condition? why not define it outside the function?

Reply:
I truly have no idea what I was thinking. Fixed it